test/PrepareDB.ts

```diff
@@ -1,14 +1,16 @@
 import { Db, MongoClient } from 'mongodb'
 
+import { loadConfigurationWithDefaults } from 'Configuration'
```

Comment:
Avoid importing things of /src from /test. There's no guarantee that `POET_INTEGRATION_NODE_URL` points to that same node.

Reply:
ok, I've added POET_MONGODB_URL.